data/com/mojang/math/SymmetricGroup3.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS com/mojang/math/SymmetricGroup3
+	METHOD <init> (Ljava/lang/String;IIII)V
+		ARG 3 first
+		ARG 4 second
+		ARG 5 third
+	METHOD compose (Lcom/mojang/math/SymmetricGroup3;)Lcom/mojang/math/SymmetricGroup3;
+		ARG 1 other
+	METHOD permutation (I)I
+		ARG 1 index
```

Comment:
Although correct, I think the mathematical term is element within the permutation set. Maybe a comment.